.github/workflows/close-stale-feature-requests.yml

```diff
@@ -0,0 +1,43 @@
+name: Close stale feature requests
+on:
+  schedule:
+    # Run every day at 1:00 AM UTC.
+    - cron: 0 1 * * *
+
+env:
+  CLOSE_MESSAGE: >
+    There has been no activity on this feature request
+    and it is being closed. If you feel closing this issue is not the
+    right thing to do, please leave a comment.
+
```

Comment:
```suggestion


```
If the goal is to have something that looks like this in the end comment (2 line breaks between the two paragraphs.)

Reply:
The goal is to have 1 space between the two paragraphs. @Mesteery are you indicating that means it needs to have two blank lines in the env devinition?

Reply:
There has been no activity on this feature request and it is being closed. If you feel closing this issue is not the right thing to do, please leave a comment.
For more information on how the project manages feature requests, please consult the [feature request management document](https://github.com/nodejs/node/blob/HEAD/doc/guides/feature-request-management.md).

---
There has been no activity on this feature request and it is being closed. If you feel closing this issue is not the right thing to do, please leave a comment.

For more information on how the project manages feature requests, please consult the [feature request management document](https://github.com/nodejs/node/blob/HEAD/doc/guides/feature-request-management.md).

---
Personally I find the second one more airy but that's just my opinion.